runtests.sh

```diff
@@ -20,16 +21,13 @@ doUnitTests=true
 
 doCodeFormatFix=false
 doBlackFormat=false
+doFlake8Format=false
 doPytypeFormat=false
 doMypyFormat=false
-doFlake8Format=false
-clean_cache_files=false
 
-NUM_PARALLEL=1
-
-# testing command to run
-cmd="python3"
+doCleanup=false
 
+NUM_PARALLEL=8
```

Comment:
Need something a bit more sophisticated here if default is > 1.  Not everyone has 8 processor to use.

NUM_PARALLEL=$(python3 -c "import multiprocessing; print(f'{multiprocessing.cpu_count()}')")

Reply:
The purpose was to have it be consistent with ``setup.cfg``.

```cfg
[pytype]
jobs = 8
```

However, ``pytype -j ${NUM_PARALLEL}`` takes precedent over ``setup.cfg`` so the value in ``setup.cfg`` doesn't matter.

Changed it back to ``1``.